SamplesDashboard/SamplesDashboard/ClientApp/src/views/Details.tsx

```diff
@@ -148,34 +184,42 @@ function renderItemColumn(item: IDetailsItem, index: number | undefined, column:
         case 'Library':
             return <span>{packageName} </span>;
 
-        case 'Used Version':
+        case 'Used version':
             return <span>{requirements} </span>;
 
-        case 'Latest Version':
+        case 'Latest version':
             return <span>{currentVersion} </span>;
 
-        case 'Azure SDK Version':
+        case 'Azure SDK version':
             return <span>{azureSdkVersion}</span>;
 
         case 'Status':
             return checkStatus(status);
     }
 }
-
+// checks the value of the status and displays the appropriate status
 function checkStatus(status: number)
 {
     switch (status) {
         case 0:
-            return <span><FontIcon iconName="StatusCircleQuestionMark" className={classNames.blue} /> Unknown </span>;
+            return <TooltipHost content="Unknown" id={'Unknown'}>
+                <span><FontIcon iconName="StatusCircleQuestionMark" className={classNames.blue} /> Unknown </span>
+            </TooltipHost>;
 
         case 1:
-            return <span><FontIcon iconName="CompletedSolid" className={classNames.green} /> Up To Date </span>;
+            return <TooltipHost content="This library is up to date" id={'UptoDate'}>
+                <span><FontIcon iconName="CompletedSolid" className={classNames.green} /> Up To Date </span>
+            </TooltipHost>;
 
         case 2:
-            return <span><FontIcon iconName="WarningSolid" className={classNames.yellow} /> Update </span>;
+            return <TooltipHost content="This library has a major/minor realease update" id={'Update'}>
+                <span><FontIcon iconName="WarningSolid" className={classNames.yellow} /> Update </span>
+            </TooltipHost>;
 
         case 3:
-            return <span><FontIcon iconName="StatusErrorFull" className={classNames.red} /> Urgent Update </span>;
+            return <TooltipHost content="This library has a patch release update" id={'UrgentUpdate'}>
```

Comment:
>This library has a patch release update [](start = 41, length = 39)

doesn't really convey why it is urgent.

Reply:
Okay, I will look into phrasing it more appropriately